content/rule_book/internationals.md

```diff
@@ -53,11 +53,6 @@ These rules apply to the Internationals quiz meet conducted once yearly, adminis
 - Elimination round will come from full International question file.
 - Finals will come from full International question file.
 
-## Tournament Structure
```

Comment:
Removing this and not putting it SOMEWHERE isn't FE right? Put in leftovers? Though not sure anyone would care? idk.

Reply:
Technically maybe. But I'd argue it's still FE since the IBQ meet direct could ignore this content whenever he feels like it.

Reply:
Fair. This is me in my "let's grease the skids for v1 getting approved" hat.

Aside from that, I have zero issues from removing it altogether. Zach isn't gonna roll with Bracket B and not tell anyone :-)

Reply:
Yup. Though for the record, I'm not concerned about Zach. I'm concerned about a future IBQ meet director who isn't Zach...

Reply:
Well, right. 


Really, the principles for running anything well are:
1. Inform completely, early, and often
2. Be open to feedback and changing (be humble)

hard to FORCE those things through policy.

Reply:
Yeah this stuff belongs better in a logistics document, not the rulebook.